docs/SUMMARY.md

```diff
@@ -20,8 +20,8 @@
 * [Creating Application](user-guide/creating-application/README.md)
   * [Git Repository](user-guide/creating-application/git-material.md)
   * [Docker Build Configuration](user-guide/creating-application/docker-build-configuration.md)
-  * [Deployment Template](user-guide/creating-application/deployment-template.md)
-    * [Rollout](user-guide/creating-application/deployment-template.md)
+  * [Deployment Template](user-guide/creating-application/deployment-template/README.md)
```

Comment:
DO NOT CHANGE this link as it is hardcoded inside Product!

Reply:
Changed to as it was